frontend/public/locales/en/translation.json

```
@@ -183,9 +183,12 @@
     "partnershipWithKleisDetail": "We received support from the technology and consulting company Kleis to help us shape our organizational processes and adopt efficient development practices. Our partnership with Kleis is extremely structuring for the development of our vision for the product and to bring perspective for foundational technical choices.",
     "openSourceContributions": "Open Source Contributions",
     "openSourceContributors": "As Tournesol is an open source project, we have been lucky to benefit from contributions by multiple volunteers. Find our wonderful contributors on <2>GitHub</2>",
-    "donateWeAreASmallTeam": "Because we are a small team of mostly volunteers, the development of Tournesol is slower than we would like it to be. If you can, please consider helping us, through coding or through donations. Check-out our <2>open source code</2>, or join our <4>Discord</4>.",
+    "donateWeAreASmallTeam": "Because we are a small team of mostly volunteers, the development of Tournesol is slower than we would like it to be. If you can, please consider helping us with a donation using one of methods below. You can also help us by contributing to our open source code. Check-out our <2>open source code</2>, or join our <5>Discord</5>.",
     "donateHowTo": "How to make a donation?",
-    "donateByDirectTransfer": "By direct transfer",
+    "donateWithUtipTitle": "With Utip",
+    "donateWithUtipDescription": "Utip is an online crowdfunding platform. Visit our <2>Utip page</2> to make a one-time or recurring donation.",
```

It seems that the official spelling is "uTip"
```suggestion
    "donateWithUtipTitle": "With uTip",
    "donateWithUtipDescription": "uTip is an online crowdfunding platform. Visit our <2>uTip page</2> to make a one-time or recurring donation.",
```